lib/surrealist/class_methods.rb

```diff
@@ -57,5 +57,41 @@ module ClassMethods
     def json_schema(&_block)
       SchemaDefiner.call(self, yield)
     end
+
+    # A DSL method to delegate schema in a declarative style. Must reference a valid
+    # class that inclues Surrealist
+    #
+    # @param [Class] klass
+    #
+    # @example DSL usage example
+    #   class Host
+    #     include Surrealist
+    #
+    #     json_schema do
+    #       { name: String }
+    #     end
+    #
+    #     def name
+    #       'Parent'
+    #     end
+    #   end
+    #
+    #   class Guest < Host
+    #     delegate_surrealization_to Host
+    #
+    #     def name
+    #       'Child'
+    #     end
+    #   end
+    #
+    #   Guest.new.surrealize
+    #   # => "{\"name\":\"Child\"}"
+    #   # For more examples see README
+    def delegate_surrealization_to(klass)
+      raise TypeError, "Expected type of Class got #{klass.class} instead" unless klass.is_a?(Class)
+      raise InvalidSchemaDelegation, 'Class does not include Surrealist' unless klass.included_modules.include?(Surrealist)
```

Comment:
this line is too long, I think, maybe we should extract it to a method?
like `raise_invalid_delegation! unless ...`
or maybe convert modifier to statement